bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -1,244 +0,0 @@
-      }
-  }
-
-  object StructurePropertyChangeListener extends PropertyChangeListener {
-    override def propertyChange(evt: PropertyChangeEvent): Unit =
-      evt.getSource match {
-        case cont: ISPObservationContainer => cont.getAllObservations.asScala.foreach(enqueue(_, 0L))
-        case _                             => // Ignore
-      }
-  }
-}
-
-object BagsManager {
-  //private val NumThreads = math.max(1, Runtime.getRuntime.availableProcessors - 1)
-  //val instance = new BagsManager(new ScheduledThreadPoolExecutor(NumThreads))
-  val instance = new BagsManager(new ScheduledThreadPoolExecutor(0))
-
-  // Check two target environments to see if the BAGS targets match exactly between them.
-  def bagsTargetsMatch(oldEnv: TargetEnvironment, newEnv: TargetEnvironment): Boolean = {
-    // Find a group with a BAGS target in it in the old and new envs.
-    val oldGroup = oldEnv.getGroups.asScala.find(gg => gg.getAll.asScala.exists(_.getBagsResult.target.isDefined))
-    val newGroup = newEnv.getGroups.asScala.find(gg => gg.getAll.asScala.exists(_.getBagsResult.target.isDefined))
-
-    // Now compare the two groups to see if they have the same BAGS targets.
-    // Filter the GuideProbeTargets of the oldGroup to make sure that we are only looking at GPTs with BAGS.
-    val oldGpt = oldGroup.toList.flatMap(gg => gg.getAll.asScala.filter(_.getBagsResult.target.isDefined))
-    val newGpt = newGroup.toList.flatMap(gg => gg.getAll.asScala.filter(_.getBagsResult.target.isDefined))
-
-    // Now compare the two lists to make sure they have the same BAGS targets.
-    (oldGpt.size == newGpt.size) &&
-      oldGpt.forall(ogpt => newGpt.exists(ngpt => ngpt.getGuider == ogpt.getGuider &&
-        ngpt.getBagsResult.target.get.getTarget.equals(ogpt.getBagsResult.target.get.getTarget)))
-  }
-
-  // Given a target environment, clear all of the BAGS targets from it.
-  def clearBagsTargets(oldEnv: TargetEnvironment): TargetEnvironment = {
-    val oldGuideEnv = oldEnv.getGuideEnvironment
-    val newGuideEnv = oldGuideEnv.getOptions.asScala.foldLeft(oldGuideEnv) { (ge, gg) =>
-      // For this guide group, iterate over all GuideProbeTargets and eliminate the BAGS targets. Filter out any
-      // GuideProbeTargets that are left empty as a result as we no longer need them.
-      val bagslessGpts = gg.getAll.asScala.map(_.withBagsResult(GuideProbeTargets.DEFAULT_BAGS_RESULT)).filter(_.containsTargets)
-
-      // If there are still any targets left, replace gg in the guide environment with a new guide group.
-      // If there are no targets left, remove gg from the guide environment.
-      if (bagslessGpts.nonEmpty) {
-        val newGG = gg.setAll(DefaultImList.create(bagslessGpts.asJavaCollection))
-        val idx = ge.getOptions.indexOf(gg)
-        ge.setOptions(ge.getOptions.updated(idx, newGG))
-      } else {
-        ge.update(OptionsList.UpdateOps.remove(gg))
-      }
-    }
-    oldEnv.setGuideEnvironment(newGuideEnv)
-  }
-}
```

Comment:
Yes... I was vacillating between commenting it out or just removing it, but I figured we can bring it back easily enough when the time comes, and there is no need to have it kicking around until then, especially since it will require fairly substantial changes.